dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "janus-idp-backstage-plugin-keycloak-backend-dynamic",
-  "version": "3.1.0",
+  "name": "backstage-community-plugin-catalog-backend-module-keycloak-dynamic",
+  "version": "3.2.0",
```

Comment:
why moving up to 3.2 and not 3.1.2?

Reply:
Had to add some changes from janus over to BCP, after that the latest version is 3.2.0

Reply:
OK!